src/interface/detele-user.interface.ts

```diff
@@ -0,0 +1,3 @@
+export interface IDeleteUser {
+  user_id: number;
```

Comment:
better to use type references like user_id: IUser['user_id'];  to keep all types referring to one value